editor/components/inserter/test/menu.js

```diff
@@ -107,12 +139,21 @@ describe( 'InserterMenu', () => {
 			/>
 		);
 
-		const activeCategory = wrapper.find( '.components-panel__body.is-opened > .components-panel__body-title' );
-		expect( activeCategory.text() ).toBe( 'Most Used' );
+		const activeCategory = wrapper.root.find( ( node ) => {
+			return node.props.className &&
```

Comment:
- When will `className` be falsey?
- This is more a perfect use case for [`classList.contains`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) (assuming `jsdom` supports it)

Reply:
Also this is prone to two issues:

- If order changes between application of `is-opened` and `components-panel__body`
- If suffixes/prefixes are applied to the class names we're testing (e.g. `not-the-components-panel__body is-opened-jk`)

Reply:
> When will className be falsey?

`find` traverses ALL components to verify there is only one match (throws an error as soon as there is more than one).  So it's possible it hits a component where `props.className` is undefined.

Good points on your subsequent comment though that need addressed.

Reply:
> This is more a perfect use case for classList.contains (assuming jsdom supports it)

`classList` is not exposed on the `node` in this iterator.  I'm a bit unclear on what you had in mind here.  Were you thinking of a different approach instead of using `TestRenderer.root.find`?

Reply:
> False negatives if order changes between application of is-opened and components-panel__body
> False positives if suffixes/prefixes are applied to the class names we're testing (e.g. not-the-components-panel__body is-opened-jk)

On re-examination, didn't those issues exist in the original test?  I'm all for improving tests but the scope of this pull was a straight conversion.  I'm open to also improving the test if thats the desire.

Reply:
Sorry, I mistook this as an HTMLElement, when in fact it's the React node. 

>On re-examination, didn't those issues exist in the original test? I'm all for improving tests but the scope of this pull was a straight conversion.

I don't think so. With the following diff applied, I'd expect the current tests to fail while the previous would have passed:

```diff
diff --git a/components/panel/body.js b/components/panel/body.js
index 1a22de279..048c3193d 100644
--- a/components/panel/body.js
+++ b/components/panel/body.js
@@ -40,7 +40,7 @@ class PanelBody extends Component {
 		const { title, children, opened, className, icon } = this.props;
 		const isOpened = opened === undefined ? this.state.opened : opened;
 		const arrow = `arrow-${ isOpened ? 'up' : 'down' }`;
-		const classes = classnames( 'components-panel__body', className, { 'is-opened': isOpened } );
+		const classes = classnames( { 'is-opened': isOpened }, 'components-panel__body', className );
 
 		return (
 			<div className={ classes }>
```

Reply:
Well, it looks like the tests were since updated, but in the original implementation it would have been an issue.

Reply:
I expect the following would still result in a false positive:

```diff
diff --git a/components/panel/body.js b/components/panel/body.js
index 1a22de279..65a86abeb 100644
--- a/components/panel/body.js
+++ b/components/panel/body.js
@@ -40,7 +40,7 @@ class PanelBody extends Component {
 		const { title, children, opened, className, icon } = this.props;
 		const isOpened = opened === undefined ? this.state.opened : opened;
 		const arrow = `arrow-${ isOpened ? 'up' : 'down' }`;
-		const classes = classnames( 'components-panel__body', className, { 'is-opened': isOpened } );
+		const classes = classnames( 'components-panel__body', className, { 'is-opened-not': isOpened } );
 
 		return (
 			<div className={ classes }>
```

Reply:
yup it would. revisiting.